extensions/standard-processors/processors/ListenTCP.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include "NetworkListenerProcessor.h"
+#include "core/logging/LoggerConfiguration.h"
+
+namespace org::apache::nifi::minifi::processors {
+
+class ListenTCP : public NetworkListenerProcessor {
+ public:
+  explicit ListenTCP(const std::string& name, const utils::Identifier& uuid = {})
+    : NetworkListenerProcessor(name, uuid, core::logging::LoggerFactory<ListenTCP>::getLogger()) {
+  }
+  ListenTCP(const ListenTCP&) = delete;
+  ListenTCP(ListenTCP&&) = delete;
+  ListenTCP& operator=(const ListenTCP&) = delete;
+  ListenTCP& operator=(ListenTCP&&) = delete;
```

Comment:
the base class `Connectable` already has these members deleted

Reply:
Good point, updated in 2798590a9068fcdf1061f4284f30e3454385ec0c